app/hooks/useDictionary.ts

```diff
@@ -49,6 +49,7 @@ export default function useDictionary() {
       em: t("Italic"),
       embedInvalidLink: t("Sorry, that link won’t work for this embed type"),
       file: t("File attachment"),
+      pdf: t("Upload PDF"),
```

Comment:
```suggestion
      pdf: t("Embed PDF"),
```

Perhaps more descriptive